README.md

```diff
@@ -9,23 +9,37 @@ We use a template repository approach for maintainability.
+
+
+1. Ensure you have the correct [visibility setting](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/setting-repository-visibility) for your GitHub account tier.
+    * If you are using GitHub from a **paid account tier** (e.g., GitHub Pro, GitHub Team, or GitHub Enterprise), you can set the visibility of this repository as either public or private; the associated GitHub Pages website will still be public.
+    * If you are using GitHub from a **free account tier** (e.g., GitHub Free for individuals or organizations), you must set the visibility of this repository as **public** for [GitHub Pages to be active](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages).
+
+2. Turn on workflow permissions for GitHub Actions.
+    * There are several [manually-triggered GitHub Actions](https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow) provided to facilitate setting up this new repository.
+    * You must turn on "Read and Write" permissions for GitHub Actions to use them:
+      * Navigate to the repository `Settings` menu by clicking the Gear icon.
+      * Navigate to the `Actions` -> `General` page from the sidebar
+      * Ensure [the following settings](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#configuring-the-default-github_token-permissions) are on:
+    <br>
+      <img src="./images/workflow-permissions.png" alt="Workflow permissions settings to turn on" width="600"/>
+
+
 
 ## Customizing the new repository for an individual training workshop
 
-We have set up a [manually-triggered GitHub Action](https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow) to support setting up this new repository.
+The first step of setting up up this repository is to run the [manually-triggered GitHub Action](https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow) "Manually trigger issue creation for standard set up" GitHub action.
 When triggered, this action will file a series of issues comprising tasks that should be completed before the start of a workshop.
-You should trigger this action once the [`training-modules` repository](https://github.com/AlexsLemonade/training-modules) tag for the given workshop has been created, as one of the _first steps_ of setting up your new repository.
-
-To run this action, navigate to the "Actions" tab.
-On the left, you will see all available workflows.
-Click the workflow named `Manually trigger issue creation for standard set up`, and then click the "Run workflow" dropdown button.
 
-## Additional instructions for externally-hosted workshops
+Run this action as follows:
 
-_More detailed instructions and/or improvements coming soon._
+* Navigate to the repository's "Actions" tab.
```

Comment:
Maybe a screenshot showing the `Actions` tab up top? Just because we previously mention navigating to (a different) `Actions` earlier when we talk about required settings.

Reply:
Actually, @Jen-OMalley let me get your opinion on how to take this screenshot. At first I was going to take in this repo, but that might look confusing since they are supposed to go to actions _in their fork_, and we can't take a screen shot of their new repo!

So, what do you think of this, just showing the bar? Or am I overthinking here?
<img width="1050" alt="Screenshot 2023-10-27 at 12 19 20 PM" src="https://github.com/AlexsLemonade/training-specific-template/assets/4701111/3f1c9e56-8147-40aa-9b9e-de4ef5282329">

Reply:
I think if we just show the bar and maybe add a few words here `Navigate to the repository's "Actions" tab at the top of the page`, that's perfectly sufficient!